### ISSUE TITLE
useNow hook causing infinite re-render??

### ISSUE THREAD

**Issue by ScottEAdams** [criipto/criipto-verify-react#12]
I am seeing some strange behaviour with re-rendering every few seconds on any page that uses result or claims with useCriiptoVerify hook. I believe the culprit is the useNow hook. Any ideas?

```
	const hasClaims = useCallback(() => {
		return claims !== null && responseType === 'token'
	}, [sessionStore, responseType, claims])
	const now = useNow(hasClaims, 3000)
```

**Comment by mickhansen** [comment]
@ScottEAdams Thanks for the report. Are you using the `sessionStore` argument?
The check inside `useCallback` should likely be changed to also check for `sessionStore` since we only want to recheck in that case.

**Comment by mickhansen** [comment]
The use case for the rerender is to clear claims once that they have expired. But yes that will cause a rerender every X seconds.

**Comment by ScottEAdams** [comment]
Yes, I am using it with sessionStore={window.sessionStorage}. I get the point of the callback its just a shame it has to cause a re-render even if there is no change.

**Comment by mickhansen** [comment]
@ScottEAdams I agree, I will see if I can find a way around that :)

**Comment by mickhansen** [comment]
@ScottEAdams potential fix in `v0.14.3` if you don't mind giving it a shot.

**Comment by ScottEAdams** [comment]
Just took it for a test spin and it seems to to do the job 👍
